meteor/client/ui/SegmentTimeline/BreakSegment.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { withTranslation } from 'react-i18next'
+import Moment from 'react-moment'
+import { getCurrentTime } from '../../../lib/lib'
+import { MeteorReactComponent } from '../../lib/MeteorReactComponent'
+import { Translated } from '../../lib/ReactMeteorData/ReactMeteorData'
+import { RundownUtils } from '../../lib/rundown'
+import { RundownTiming } from '../RundownView/RundownTiming/RundownTiming'
+
+interface IProps {
+	breakTime: number | undefined
+}
+
+interface IState {
+	displayTimecode: number | undefined
+}
+
+class BreakSegmentInner extends MeteorReactComponent<Translated<IProps>, IState> {
+	constructor(props: IProps) {
+		super(props)
+
+		this.state = {
+			displayTimecode: undefined,
+		}
+
+		this.updateTimecode = this.updateTimecode.bind(this)
+	}
+
+	componentDidMount() {
+		window.addEventListener(RundownTiming.Events.timeupdate, this.updateTimecode)
```

Comment:
Why use the event here?

Reply:
Changed to use `withTiming` https://github.com/tv2/tv-automation-server-core/commit/f35d71bb8b22a29c9a9a494f4068b1da247309ca